src/BuildTasks/RoslynTools.BuildTasks.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard1.3</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <Description>RoslynTools build tasks</Description>
+    <PackageTags>RoslynTools msbuild task</PackageTags>
+    <IsTool>true</IsTool>
+
+    <!-- Using an explicit nuspec file since NuGet Pack target currently doesn't support including dependencies in tools packages -->
+    <IsPackable>true</IsPackable>
```

Comment:
The IsPackable property si duplicated in this file

Reply:
Thanks, will clean up.